.github/workflows/update-compat-data.yml

```diff
@@ -14,8 +14,8 @@ jobs:
     permissions:
       contents: write # for Git to git push
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
+      - uses: actions/checkout@v4
```

Comment:
```suggestion
      - uses: actions/checkout@v4
```

Reply:
The first action is to checkout Babel repo and the second one is to checkout `compat-table`.